hotelReservation/services/review/proto/review.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package review;
+option go_package="github.com";
```

Comment:
Is this pointing to the right directory? I would've thought it would be the following: `github.com/delimitrou/DeathStarBench/hotelreservation/services/review`

Reply:
Fixed